src/export/ScriptExporter.java

```diff
@@ -5,13 +5,11 @@
 import map.*;
 
 public strictfp class ScriptExporter {
-	
-	private static DataOutputStream out;
 
-	public static void exportScript(String folderPath, String mapname, SCMap map) throws IOException {
+    public static void exportScript(String folderPath, String mapname, SCMap map) throws IOException {
 		File file = new File(folderPath + mapname + File.separator + mapname + "_script.lua");
 		file.createNewFile();
-		out = new DataOutputStream(new BufferedOutputStream(new FileOutputStream(file)));
+        DataOutputStream out = new DataOutputStream(new BufferedOutputStream(new FileOutputStream(file)));
```

Comment:
should these stay as static fields so they can be accessed from methods called by `exportScript` without passing references around?